packages/client/src/components/Card.tsx

```diff
@@ -22,45 +21,30 @@ function name([value, suit]: CardType) {
   return `${VALUES[value]} of ${SUIT[startCase(suit)]}`;
 }
 
-const suitList = Object.values(SUIT);
-function bgPosition([value, suit]: CardType) {
-  return `${(value - 1) * 11}% ${suitList.indexOf(suit) * 33}%`;
+function imgUrl([value, suit]: CardType) {
+  return require(`../images/${suit.toLowerCase()}/${padStart(`${value}`, 2, '0')}.jpg`).default;
```

Comment:
Could you use `import` here? Manually construct the name as you do, import it, and return it.

Reply:
I don't think you can import dinamically, this is a weird case, we will replace with links once we have a cdn but for now it should be ok with require